frameworks/supervised/exec.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import os
+import tempfile as tmp
+import pandas as pd
+import numpy as np
+
+from frameworks.shared.callee import call_run, result, Timer, touch
+
+log = logging.getLogger(os.path.basename(__file__))
+
+import supervised
+from supervised.automl import AutoML
+import shutil
+
+
+def make_subdir(name, config):
+    subdir = os.path.join(config.output_dir, name, config.name, str(config.fold))
+    touch(subdir, as_dir=True)
+    return subdir
+
+def run(dataset, config):
+    log.info("\n**** mljar-supervised ****\n")
+
+    is_classification = config.type == "classification"
+
+    X_train, X_test = dataset.train.X, dataset.test.X
+    y_train, y_test = dataset.train.y, dataset.test.y
+
+    X_train = pd.DataFrame(X_train, columns=[f"f{i}" for i in range(X_train.shape[1])])
+    X_test = pd.DataFrame(X_test, columns=[f"f{i}" for i in range(X_test.shape[1])])
```

Comment:
if, for some reason, you want the real column names, see the integration for AutoGluon here: https://github.com/openml/automlbenchmark/pull/109/files#diff-afee22ac53339a4a6051bba646ad17c3

Reply:
Yep, changed!